setup/azurecmd.wxs

```diff
@@ -1,119 +1,87 @@
                   DowngradeErrorMessage="A later version of Microsoft Azure PowerShell is already installed. Setup will now exit." />
 
     <Icon Id="icon.ico" SourceFile="media\WAIcon.ico"/>
+
     <Property Id="ARPPRODUCTICON" Value="icon.ico" />
     <Property Id="ALLUSERS" Value="1" />
     <Property Id="ISVSINSTALL" Secure="yes"/>
+    <Property Id="POWERSHELLEXE">c:\windows\system32\WindowsPowerShell\v1.0\powershell.exe</Property>
+
+    <WixVariable Id="WixUILicenseRtf" Value="$(sys.SOURCEFILEDIR)\media\License.rtf" />
+    <UIRef Id="WixUI_Minimal" />
+    <WixVariable Id="WixUIBannerBmp" Value="media\bannrbmp.bmp" />
+    <WixVariable Id="WixUIDialogBmp" Value="media\dlgbmp.bmp" />
+
     <!-- per machine install-->
     <Property Id="PSCOMPATIBLEVERSION">
       <RegistrySearch Id="PSCOMPATIBLEVERSION" Root="HKLM" Key="SOFTWARE\Microsoft\PowerShell\3\PowerShellEngine" Name="PSCompatibleVersion" Type="raw" />
     </Property>
 
-    <SetProperty Action="SetBase64" Id="BaseModulesFolder" Value="[ProgramFiles64Folder]WindowsPowerShell\Modules\" Before="AppSearch">
-      VersionNT64
-    </SetProperty>
-
-    <SetProperty Action="SetBase32" Id="BaseModulesFolder" Value="[ProgramFilesFolder]WindowsPowerShell\Modules\" Before="AppSearch">
-      NOT VersionNT64
-    </SetProperty>
-
-    <SetProperty Id="POWERSHELLEXE" Value="[WindowsFolder]System32\WindowsPowerShell\v1.0\powershell.exe" Before="AppSearch"/>
-    
-    <util:CloseApplication Id="ClosePowerShellExe" Target="powershell.exe" ElevatedCloseMessage="yes" CloseMessage="yes" PromptToContinue="yes" Description="PowerShell must be closed to continue with this installation.  Please close any PowerShell windows and click 'Retry', or click 'Cancel' to stop installation. If you click 'Ignore' the installer will attempt to close any open PowerShell window."/>
-    <util:CloseApplication Id="ClosePowerShellIseExe" Target="powershell_ise.exe" ElevatedCloseMessage="yes" CloseMessage="yes" PromptToContinue="yes" Description="PowerShell ISE must be closed to continue with this installation. Please close any PowerShell ISE windows and click 'Retry', or click 'Cancel' to stop installation. If you click 'Ignore' the installer will attempt to close any open PowerShell ISE window."/>
-
-    <Condition Message="This setup requires the Windows PowerShell 3.0 or compatible version to be installed.">
-      <!-- This condition will break if PowerShell has version x3.0 and it is not compatible with 3.0 -->
-      <![CDATA[Installed or (PSCOMPATIBLEVERSION >< "3.0")]]>
-    </Condition>
-
-    <PropertyRef Id="NETFRAMEWORK45" />
-    <Condition Message="This setup requires the .NET Framework 4.5.2 or higher to be installed.">
-      <!-- NETFRAMEWORK45 contains a DWORD with installed version key to versions is here: -->
-      <!-- https://msdn.microsoft.com/en-us/library/ee942965(v=vs.110).aspx#detect_net -->
-      <![CDATA[Installed or (NETFRAMEWORK45 >= "#379893") or ISVSINSTALL]]>
```

Comment:
I believe that we still need to check .Net Framework version.  LMK if you verified that PowerShell 5 + .Net < 4.5.2 is possible

Reply:
Is it possible to set up PowerShell 5 to run with .Net < 4.5.2? Scott and I could not get PowerShell to run without 4.5.2 installed.